Runtimes/Supplemental/cmake/modules/FindSwiftCore.cmake

```diff
@@ -0,0 +1,111 @@
+#[=======================================================================[.rst:
+FindSwiftCore
+------------
+
+Find SwiftCore, deferring to SwiftCoreConfig.cmake when requested.
+This is meant to find the core library to be linked by the Supplemental libraries.
+
+Imported Targets
+^^^^^^^^^^^^^^^^
+
+The following :prop_tgt:`IMPORTED` TARGETS may be defined:
+
+ ``SwiftCore``
+
+Hint Variables
+^^^^^^^^^^^^^^
+
+ ``SDKROOT``
+   Set the path to the Swift SDK Root.
+   This only affects MacOS builds.
+
+ ``swift_SDKROOT``
+   Set the path to the Swift SDK installation.
+   This only affects Linux and Windows builds.
+   Apple builds always use the library provided by the SDK.
+
+Result Variables
+^^^^^^^^^^^^^^^^
+
+The module may set the following variables if `SwiftCore_DIR` is not set.
+
+ ``SwiftCore_FOUND``
+   true if core was found
+
+ ``SwiftCore_LIBRARIES`` OR ``SwiftCore_IMPLIB``
+   the libraries to be linked
+
```

Comment:
`SwiftCore_INCLUDE_DIR` is also set.

Reply:
This can also be a property on the imported library target.